IDE/GCC-ARM/Makefile.common

```diff
@@ -113,18 +114,28 @@ SRC_C += ../../wolfcrypt/src/wolfcrypt_last.c
 endif
 
 # wolfCrypt Additional
+SRC_C += ../../wolfcrypt/src/asm.c
+SRC_C += ../../wolfcrypt/src/async.c
```

Comment:
A direct clone from GitHub will not have this file and only make dist or packaged bundles will have the async.c file.

Reply:
wrapped